server/routes/find-location.js

```diff
@@ -17,11 +17,11 @@ module.exports = [{
     } else {
       // Error
       const err = request.yar.get('displayError')
-      request.yar.set('displayError', {})
-      const location = request.yar.get('locationError').input
-      request.yar.set('locationError', {})
+      request.yar.set('displayError', null)
+      const location = request.yar.get('locationError').input || ''
```

Comment:
This was overriding the location error into a 500 error because the `input` wasn't always being set, so coalesce it to nothing